.gitignore

```diff
@@ -26,3 +26,5 @@ node_modules
 
 # app config file
 .env
+
+Dockerfile
```

Comment:
🤔

Reply:
Yup because you have two ways now and if you modify Dockerfile you'll have issues by pulling afterwards.